network-api/networkapi/wagtailpages/pagemodels/research_hub/detail_page.py

```diff
@@ -134,8 +136,15 @@ def get_context(self, request):
         context = super().get_context(request)
         context["breadcrumbs"] = self.get_breadcrumbs()
         context["authors_index"] = authors_index.ResearchAuthorsIndexPage.objects.first()
+        context["research_authors"] = self.get_research_authors()
         return context
 
+    def get_research_authors(self):
+        research_author_profiles = localize_queryset(
+            Profile.objects.filter(authored_research__research_detail_page=self)
```

Comment:
I think we need a `prefetch_related` here to avoid the N+1 problem, something like:

```python
Profile.objects.prefetch_related("authored_research").filter(authored_research__research_detail_page=self)
```
Otherwise, for each `authored_research` in a `Profile`, we will have additional queries.

Reply:
Hey Jhonatan, thanks for walking through this comment with me over zoom. I have updated the PR to now include the `prefetch_related` method!